spec/models/employer_events/employer_importer_spec.rb

```diff
@@ -148,6 +148,112 @@
       end
     end
   end
+
+describe "with published plan years and carrier ids" do
+  let(:first_plan_year_start_date) { Date.new(2017, 4, 1) }
+  let(:first_plan_year_end_date) { Date.new(2018, 3, 31) }
+  let(:last_plan_year_start_date) { Date.new(2018, 4, 1) }
+  let(:last_plan_year_end_date) { Date.new(2019, 3, 31) }
+  let(:employer_event_xml) do
+		<<-XMLCODE
+		<plan_years xmlns="http://openhbx.org/api/terms/1.0">
+			<plan_year>
+				<plan_year_start>#{first_plan_year_start_date.strftime("%Y%m%d")}</plan_year_start>
+				<plan_year_end>#{first_plan_year_end_date.strftime("%Y%m%d")}</plan_year_end>
+				<open_enrollment_start>20151013</open_enrollment_start>
+				<open_enrollment_end>20151110</open_enrollment_end>
+				<benefit_groups>
+					<benefit_group>
+						<name>Health Insurance</name>
+						<elected_plans>
+							<elected_plan>
+								<id>
+									<id>A HIOS ID</id>
+								</id>
+								<name>A PLAN NAME</name>
+								<active_year>2015</active_year>
+								<is_dental_only>false</is_dental_only>
+								<carrier>
+									<id>
+										<id>20011</id>
+									</id>
+									<name>A CARRIER NAME</name>
+								</carrier>
+							</elected_plan>
+							<elected_plan>
+								<id>
+									<id>A HIOS ID</id>
+								</id>
+								<name>A PLAN NAME</name>
+								<active_year>2015</active_year>
+								<is_dental_only>false</is_dental_only>
+								<carrier>
+									<id>
+										<id>20012</id>
+									</id>
+									<name>A CARRIER NAME</name>
+								</carrier>
+							</elected_plan>
+						</elected_plans>
+					</benefit_group>
+				</benefit_groups>
+       </plan_year>
+     </plan_years>
+		XMLCODE
+  end
```

Comment:
I checked your source xml and it has the right format - we should add a couple more higher root nodes maybe though so it reflects the actual document tree - if you notice my previous specs actually cheat a little bit and I'd prefer you not follow that pattern.